shared/src/helpers/convert-input-to-date/convert-input-to-date.ts

```diff
@@ -0,0 +1,15 @@
+import { getDateEnding } from '../helpers';
+
+const convertInputToDate = (value: number): string => {
+    const monthInYear = 12;
+    const yearRounder = 10;
+    const dateTitle = value < monthInYear ? 'month' : 'year';
+    const dateValue =
+        value < monthInYear
+            ? value
+            : Math.round((value / monthInYear) * yearRounder) / yearRounder;
+    const refactoredDateTitle = getDateEnding(dateTitle, dateValue);
+    return `${dateValue} ${refactoredDateTitle}`;
+};
```

Comment:
For now, we don't know if there will be a possibility to add a month, you can keep this function somewhere if you don't want to lose it, but for now, we don't need it and it's better to remove it. We'll also use day js lib to work with dates so it may be easier in future

Reply:
fix